src/ansible_navigator/__init__.py

```diff
@@ -1,2 +1,14 @@
-"""The ansible-navigator application."""
-from ._version import __version__  # noqa: F401
+"""The ansible-navigator application.
+
+This is a transitional configuration during the migration
+from the hardcoded __version__ in _version to
+version being populated during the build.
+"""
+
+try:
+    from ._version import __version__
+except ImportError:
+    try:
+        from ._version import version  # type: ignore[attr-defined]
+    except ImportError:
+        __version__ = "source"
```

Comment:
I don't think this is the right value for the fallback. In source checkouts, even with an editable install mode, this won't happen.

Reply:
Plus, if the second import succeeds, it'll cause a `NameError` in other components.

Reply:
this will catch `python -m ansible_navigator` and return the version as `source`

Reply:
That scenario should be covered by calling the version detection helper from setuptools-scm.

Reply:
Although, it's probably not the best idea to support it because if it's not installed, the import machinery will fail to import the missing deps long before this variable will get used. And if it's installed, then it won't cause an ImportError. This line is probably unreachable.